sw/device/silicon_creator/mask_rom/mask_rom.c

```diff
@@ -73,11 +74,14 @@ static rom_error_t mask_rom_init(void) {
   pinmux_init();
   // Configure UART0 as stdout.
   uart_init(kUartNCOValue);
-  // Initialize the shutdown policy according to lifecycle state.
+
   lc_state = lifecycle_state_get();
+  // Re-initialize the watchdog timer.
+  watchdog_init(lc_state);
+  // Initialize the shutdown policy.
```

Comment:
We need a `SEC_MMIO_WRITE_INCREMENT()` here.

@moidx added that macro and `SEC_MMIO_ASSERT_WRITE_INCREMENT()` in #10552. You can define constants in `watchdog.h` similar to those in that PR, e.g. `kWatchdogSecMmioInit = X`, then use `SEC_MMIO_WRITE_INCREMENT(kWatchdogSecMmioInit);` here and `SEC_MMIO_ASSERT_WRITE_INCREMENT(kWatchdogSecMmioInit, X)` in `watchdog_init()` (and other functions that use sec_mmio).

Reply:
Thanks, done.